### PR TITLE
SC | Fix uploaded file focus management

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > After uploading a file in the Supplemental Claim form, the last uploaded card would scroll into view but focus would be lost (moved to the document `body`). What we want is for the document type select to receive focus. This PR:
  > - Removes the `focusUploads` function because it interferes with `focusFileCard` which is called after the upload response is parsed.
  > - Updated `focusFileCard` to properly use the `focusElement` function to wait for the web component shadow DOM to render before searching for and focusing on the `select` within the `va-select` web component.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
  > The `focusFileCard` was already working as expected for the NOD upload page
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#83632](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83632)

## Testing done

- _Describe what the old behavior was prior to the change_
  > Focus on uploaded file document type select was lost
- _Describe the steps required to verify your changes are working as expected_
  > Test uploads in review instance or locally
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Focus lost to focusing on select | uploads-before | uploads |

## What areas of the site does it impact?

Supplemental Claim uploads page

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/shared/utils/focus.js [hunk 1]
```diff
@@ -95,6 +100,11 @@ export const focusRadioH3 = () => {
   }
 };
 
+export const focusH3 = () => {
+  scrollTo('topContentElement');
+  focusByOrder(['#main h3', defaultFocusSelector]);
+};
+
```
Comment (shaunburdick) [inline src/applications/appeals/shared/utils/focus.js:107]: Should this portion be part of #30064?

Comment (Mottie) [reply src/applications/appeals/shared/utils/focus.js:107]: It is, but I needed to add it to make this PR work. I'll update this branch and this change will disappear.

**Review by shaunburdick — APPROVED** [review]
LGTM

File: src/applications/appeals/shared/utils/focus.js (hunk 1 above)
Comment (shaunburdick) [reply src/applications/appeals/shared/utils/focus.js]: Noice!

**Pull request merged**